docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,151 @@
+# Onboarding to a cluster
+
+Currently we support 2 ways to request/perform onboarding to any supported clusters:
+
+1. Onboarding request - please use the **Onboarding to Cluster** issue template in [operate-first/support](https://github.com/operate-first/support) repository.
+2. Opening a PR with the desired changes. The rest of this doc focuses on this second option. We'd gladly welcome your contributions.
+
+## Prerequisites
+
+1. Please install our [toolbox](https://github.com/operate-first/toolbox) to have the developer setup ready automatically for you.
+2. Or you can install the tools manually. You'll need [kustomize](https://kustomize.io/), [sops](https://github.com/mozilla/sops) and [ksops](https://github.com/viaduct-ai/kustomize-sops).
+
+Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. **During this whole setup, we'll be working within this repository.**
+
+## Recipe
+
+If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).
+
+In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.
+
+## Adding namespaces
+
+### Base resources
+
+Please run:
+
+```sh
+bash hack/onboarding.sh NAMESPACE_NAME OWNER_TEAM OPTIONAL_PROJECT_DESCRIPTION
+```
+
+This script will create
+
+- A namespace in `cluster-scope/base/namespaces/$NAMESPACE_NAME`
+- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/groups/$OWNER_TEAM`
+- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.
+
+### Enabling namespace deployment to a specific cluster
+
+```sh
+cd cluster-scope/overlays/TARGET_CLUSTER
+kustomize edit add resource ../../base/namespaces/$NAMESPACE
+```
+
+## Adding users
```

Comment:
Can we add a one liner that says something like the following: 

"The following steps enable users to access designated cluster/namespaces. This simply consists of adding such users to the appropriate OpenShift groups with access to those cluster/namespaces. These groups are encrypted to protect user sensitive data."

Reply:
Copy pasted with a slight changes to the PR. :smile: